### ISSUE TITLE
In how far does it differentiate itself from Wikipedia?

### ISSUE THREAD

**Issue by ErfolgreichCharismatisch** [uprtcl/spec#3]
See title.

**Comment by pepoospina** [comment]
- Wikipedia is a single application. _Prtcl is a protocol, so content can be spread into multiple locations, including web3 networks.
- _Prtcl content is stored as JSON objects, with any structure, Wikipedia's content uses it's own formatted text standard.
- Each page/object can be decomposed into subobject in _Prtcl. In wikipedia a page cannot have subpages.

**Comment by ErfolgreichCharismatisch** [comment]
So compatibility and expandability.
Yet, what is the average joe's incentive to use it?

**Comment by pepoospina** [comment]
On one hand, the GIT-like experience: Having multiple versions of the same object. We think this is very useful on some environments like mutliple parties working on the same document, or letting people propose changes to content without having write permission.

And then, compatibility and expandability will enable a wide set of interoperable applications, making old siloed applications feel dumb.

**Comment by ErfolgreichCharismatisch** [comment]
The first idea that came to mind was the official narrative regarding Covid and alternative narratives.

Let's assume a structure of

**Government**

- Assumption 1
- Conclusion 1

- Assumption 2
- Conclusion 2

- Assumption 3
- Conclusion 3

Would the average joe now fork the government approach and change Conclusion 1 or Assumption 3 to his perspective?

**Comment by pepoospina** [comment]
Yes, then make a pull request. We could even replace the Government with a DAO to accept the PR. 
Have you seen the DAO-governed content in the [demos](https://www.notion.so/uprtcl/Demos-84b677eeab7a445bbb274585ff72ffba)?

**Comment by ErfolgreichCharismatisch** [comment]
What is a DAO, what a PR?

**Comment by pepoospina** [comment]
Pull request.
DAOs are "democracies on the blockchain".

**Comment by ErfolgreichCharismatisch** [comment]
What I see in your work is a big focus on avoiding censorship(blockchain approach), which seems to be the biggest difference to other approaches.

I wonder: was that developed based on a requirement or your fascination for it?

**Comment by pepoospina** [comment]
We work with different blockchain communities, they wanted to have DAO-governed content and gave us some funds.

Hop on to our discord and we can continue talking there :)
https://discord.gg/B9RNY9s

**Comment by nborson** [comment]
That's a great question @ErfolgreichCharismatisch . I'm just learning about the Underscore Protocol myself, but I think _Prtcl could make Wikipedia even better by composing articles using standard structured objects ([evees](https://www.uprtcl.io/)) for its paragraphs and sections.
- The "main branch" of these objects could be formally and transparently controlled through a [democratic process](https://www.notion.so/Intercreativity-Web3-Governance-6d4b71158fde4129a4b60cd8d6e4b318), while other perspectives could flourish alongside, depending less on individual editors who might not be completely "objective," if there is such a thing.
- Objects would link to each other [bi-directionally](https://maggieappleton.com/bidirectionals) not just across articles, but between platforms, sites, and apps, so Wikipedia information would be much easier to re-use, connect, grow, and evolve.

But as @pepoospina said, Wikipedia is a single application, so it is just one possible use for _Prtcl.
- An important feature of _Prtcl is access control at the object level, so it can be used for private as well as public information.
- Evees will be used to build personal knowledge management applications and connect and share among them.
- _Prtcl will connect evees from all kinds of apps and platforms; they could be used in Facebook, Twitter, news sites... everywhere! This is what the creator of the World-Wide Web [called](https://www.w3.org/People/Berners-Lee/1997/w3j-3-iview.html) Intercreativity.
